### ISSUE TITLE
[Style] Enhance UI of instructions

### ISSUE THREAD

**Issue by tarnibalgoher5** [Rakesh9100/Click-The-Edible-Game#225]
### What's the style idea?

We need to improve the instruction manual ,it looks a bit wordy which withdraws user attention after just a few seconds.

It can be enhanced as follows:
-Add cards for each instruction
-Only the main keywords are kept in bold and increased font, the extra information is mentioned as description on each card in a smaller font size   

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Click-The-Edible-Game/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [ ] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [X] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Click-The-Edible-Game/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by Rakesh9100** [comment]
Proceed @tarnibalgoher5

**Comment by Rakesh9100** [comment]
Status of your PR @tarnibalgoher5

**Comment by tarnibalgoher5** [comment]
I'm designing the cards, there's a little issue I'm encountering with the overall alignment of the instructions section, will try to fix it by tomorrow and update you further.

**Comment by tarnibalgoher5** [comment]
Yes, working on the design

**Comment by harshvardhan614** [comment]
@Rakesh9100 can I work on this issue?

**Comment by Rakesh9100** [comment]
Issue will be closed if PR is not up by today @tarnibalgoher5
